stl/inc/yvals_core.h

```diff
@@ -799,7 +800,19 @@
 #define _DEPRECATE_STDEXT_HASH_UPPER_BOUND
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4024

```

Comment:
FYI, we'll likely be applying the `string::reserve` PR (#176) today - @StephanTLavavej has written up some test coverage - which also adds a deprecation here, so you will probably need to rebase and resolve a merge conflict.

Reply:
Waiting for merge #176.